pkg/security/probe/probe.go

```diff
@@ -770,6 +777,15 @@ func (p *Probe) ApplyRuleSet(rs *rules.RuleSet, dryRun bool) (*Report, error) {
 					}
 					log.Debugf("failed to register kProbe `%s`", kprobe.Name)
 				}
+				if len(hookPoint.Tracepoint) > 0 {
```

Comment:
How about adding `&& err == nil` in the condition you highlighted + use `err = p.Module.RegisterTracepoint(hookPoint.Tracepoint)` instead of `err := p.Module.RegisterTracepoint(hookPoint.Tracepoint)` in the following line ?

We actually need the error check at line 790 for tracepoints as well.